### PR TITLE
a russian

### PR DESCRIPTION
**Pull request by boodaliboo**
#### Changelog

:cl:  
tweak: tweaks russian simple mobs name to make more sense when examined
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
image
this is what it shows currently

File: code/modules/mob/living/simple_animal/hostile/russian.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /mob/living/simple_animal/hostile/russian
-	name = "Russian"
+	name = "A Russian"
```
Comment (Bibby0110) [inline code/modules/mob/living/simple_animal/hostile/russian.dm]:
```suggestion
	name = "\improper Russian"
```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

**Comment by boodaliboo** [comment]
does /improper just tell the game its not a proper noun

**Comment by Ratvar** [comment]
уот так уот

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**